alf/networks/actor_distribution_networks.py

```diff
@@ -43,7 +43,7 @@ def __init__(self,
             conv_layer_params (list[tuple]): a list of tuples where each
```

Comment:
Should search for all `fc_layer_params` and `conv_layer_params` to change them to tuple

Reply:
Done